nemo/collections/tts/models/glow_tts.py

```diff
@@ -242,6 +252,27 @@ def setup_validation_data(self, val_data_config: Optional[DictConfig]):
     def setup_test_data(self, test_data_config: Optional[DictConfig]):
         self._test_dl = self._setup_dataloader_from_config(cfg=test_data_config)
 
+    def generate_spectrogram(self, text, noise_scale=0.0, length_scale=1.0):
+
+        self.eval()
+
+        text_parser = self.get_parser()
+
+        if text[-1] != ".":
+            text = text + "."
+
+        text_seq = text_parser(text)
+
+        text_seq = [11] + text_seq + [11]
```

Comment:
??? Why 11? Is that a space or pad or <BOS> or <EOS>?

Reply:
removed this line